nvme_gw.config

```diff
@@ -12,10 +12,14 @@
 enable_auth = False
 gateway_addr = [::]
 gateway_port = 5500
-spdk_path = /path/to/spdk
-spdk_tgt = spdk/build/bin/nvmf_tgt
+gateway_group =
 grpc_server_max_workers = 10
 
+[ceph]
+
+ceph_pool = rbd
+ceph_config_file = /etc/ceph/ceph.conf
```

Comment:
Actually the same note about `ceph_` prefix for params in `[ceph]` section. They seem not necessary. We could have:
```
[ceph]
config_file = /etc/ceph/ceph.conf
pool = rbd
```
Alternatively, the section could be named `[rbd]` (because it is used for "rbd bdev"):
```
[rbd]
ceph_config_file = /etc/ceph/ceph.conf
pool = rbd
```